<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>site2site</class>
 <widget class="QWidget" name="site2site">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;h1&gt;Site to site VPN (IPsec)&lt;/h1&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="enableVPN">
     <property name="text">
      <string>Enable site to site VPN service</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0" colspan="2">
        <widget class="QTableView" name="VPNtable">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>452</width>
           <height>29</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="addVPN">
           <property name="text">
            <string>New</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../../nucentral-edenwall/resources/edenwall.qrc">
             <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delVPN">
           <property name="text">
            <string>Delete</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../../nucentral-edenwall/resources/edenwall.qrc">
             <normaloff>:/icons-32/moins.png</normaloff>:/icons-32/moins.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;h2&gt;Configured VPN(s)&lt;/h2&gt;</string>
         </property>
         <property name="buddy">
          <cstring>VPNtable</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&lt;h2&gt;EdenWall Appliance fingerprint&lt;/h2&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="buddy">
          <cstring>myfingerprint</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="myfingerprint">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>64</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>64</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByMouse</set>
         </property>
         <property name="backgroundVisible">
          <bool>false</bool>
         </property>
         <property name="centerOnScroll">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>ClosedHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Copy to clipboard</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../../nucentral-edenwall/resources/edenwall.qrc">
           <normaloff>:/icons-20/copy.png</normaloff>:/icons-20/copy.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0" colspan="2">
        <widget class="QTableView" name="RSAtable">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>452</width>
           <height>29</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="addRSA">
           <property name="text">
            <string>New</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../../nucentral-edenwall/resources/edenwall.qrc">
             <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delRSA">
           <property name="text">
            <string>Delete</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../../nucentral-edenwall/resources/edenwall.qrc">
             <normaloff>:/icons-32/moins.png</normaloff>:/icons-32/moins.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&lt;h2&gt;Keys fingerprints&lt;/h2&gt;</string>
         </property>
         <property name="buddy">
          <cstring>RSAtable</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionRefresh_VPN_states">
   <property name="icon">
    <iconset resource="../../../../../nucentral-edenwall/resources/edenwall.qrc">
     <normaloff>:/icons/refresh.png</normaloff>
     <normalon>:/icons/refresh.png</normalon>:/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh site to site VPN states</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../../nucentral-edenwall/resources/edenwall.qrc"/>
 </resources>
 <connections/>
</ui>
